lib/actions/reassemble.js

```diff
@@ -6,6 +6,16 @@ let timeHandle;
 let groupList = [];
 let delay;
 
+function randonGUID() {
+  return 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, (c) => {
+    // eslint-disable-next-line no-bitwise
+    const r = Math.random() * 16 | 0;
+    // eslint-disable-next-line no-mixed-operators
+    const v = c === 'x' ? r : (r && 0x3 || 0x8);
+    return v.toString(16);
+  });
+}
+
```

Comment:
Can you use [a library for this](https://www.npmjs.com/package/uuid)?

Reply:
Yes Sure!